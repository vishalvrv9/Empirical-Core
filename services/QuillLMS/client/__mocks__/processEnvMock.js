export default {
  env: {
    CDN_URL: 'http://cdn.quill.org'
  }
}
