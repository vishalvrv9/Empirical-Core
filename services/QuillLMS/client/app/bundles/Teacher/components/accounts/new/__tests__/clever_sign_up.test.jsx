import React from 'react';
import { shallow } from 'enzyme';

import CleverSignUp from '../clever_sign_up'

import processEnvMock from '../../../../../../../__mocks__/processEnvMock.js';
window.process = processEnvMock;

describe('CleverSignUp component', () => {

  it('should render', () => {
    const wrapper = shallow(
        <CleverSignUp />
    );
    expect(wrapper).toMatchSnapshot();
  });

});
