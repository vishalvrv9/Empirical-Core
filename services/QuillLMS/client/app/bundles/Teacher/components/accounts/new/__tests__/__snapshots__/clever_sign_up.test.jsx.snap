// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CleverSignUp component should render 1`] = `
<a
  className="clever-sign-up"
  href=""
>
  <img
    src="http://cdn.quill.org/images/shared/clever_icon.svg"
  />
  <span>
    Sign up with Clever
  </span>
</a>
`;
